src/test/java/website/frontrow/util/CollisionHandlerTest.java

```diff
@@ -114,9 +114,9 @@ public void slightlyOccupiedAABBTest()
                 Cell.WALL, Cell.EMPTY, Cell.WALL,
                 Cell.WALL, Cell.WALL, Cell.WALL), 3, 3));
         CollisionHandler handler = new CollisionHandler(check);
-        assertTrue(handler.checkLevelAABB(
-                new AABB(new Point(1.5, 1.5), new Point(2.5, 2.5)), new Point(0, 0)
-        ));
+        Cell cell = handler.checkLevelAABB(
+                new AABB(new Point(1.5, 1.5), new Point(2.5, 2.5)), new Point(0, 0));
+        assertFalse(cell.getType() == Cell.EMPTY);
```

Comment:
Use assertNotEquals instead of == operator combined with assertFalse, == does not work properly for non-objects.
<testing